api/server/server.go

```diff
@@ -177,7 +177,7 @@ func (s *Server) makeHTTPHandler(handler httputils.APIFunc) http.HandlerFunc {
 // InitRouters initializes a list of routers for the server.
 func (s *Server) InitRouters(d *daemon.Daemon) {
```

Comment:
maybe we should remove this function in the future, or group all backends into a single `APIBackend`.